From: "Stefan Dösinger" <stefan@codeweavers.com>
Subject: [PATCH] configure: Mark CROSSCC precious.
Message-Id: <20220411110447.320820-1-stefan@codeweavers.com>
Date: Mon, 11 Apr 2022 14:04:47 +0300

So that configure remembers e.g. ccache for building PE files.

Signed-off-by: Stefan Dösinger <stefan@codeweavers.com>

---

One cache to rule them all, one hash to find them, one cache to bring
them all and in .ccache bind them. In the land of $HOME where the disk
is full.

I probably spent more time making fun about "precious" than figuring out
all this autoconf stuff, so maybe a better solution is needed.
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 11111111111..11111111111 100644
--- a/configure.ac
+++ b/configure.ac
@@ -809,6 +809,7 @@ enable_wineandroid_drv=${enable_wineandroid_drv:-no}
 enable_winemac_drv=${enable_winemac_drv:-no}
 
 dnl Check for cross compiler(s)
+AC_ARG_VAR(CROSSCC, "Cross compiler for PE files")
 AC_SUBST(PE_ARCHS,"")
 cross_archs=
 AS_VAR_SET_IF([enable_archs],

-- 
2.40.0

